### ISSUE TITLE
Collections issues

### ISSUE THREAD

**Issue by hayleyorndorf** [qubeshub/hubzero-cms#82]
Collections text is not contrasated enough with gray background. "Image not found" issues - user/dev related or something more significant?
![collections](https://user-images.githubusercontent.com/25186673/36110033-486c0684-0fef-11e8-8d38-5a6c86c1db23.PNG)

**Comment by mdlama** [comment]
Yep, this contrast issue is on my "increase contrast" throughout the site list - thanks for pointing it out.  

The image not found thing is dev related - shouldn't be an issue on production (famous last words...)
